docs/tutor.rst

```diff
@@ -68,9 +68,18 @@ You can then browse the documentation with::
 Releasing a new version
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-- Bump the ``__version__`` value in ``tutor/__about__.py``. (see :ref:`versioning` below)
-- Collect changelog entries with ``make changelog``.
-- Create a commit with the version changelog.
+Releasing a version includes two phases:
+
+1. Add changes and generate individual changelog entries:
+
+- run ``make changelog-entry``(or ``scriv create``) command - It will create changelog entries in a folder named changelog.d for the changes user has done for releasing a new version.
+- Commit and merge all the changes including the changelog entries. e.g this `commit <https://github.com/overhangio/tutor-discovery/commit/e30a78936d63439bde069aeff11960585bd81592>`_.
```

Comment:
The "commit" links is causing the documentation build test to fail: https://github.com/overhangio/tutor/actions/runs/7622960135/job/20762024650?pr=991

You should use anonymous links, with double "_": https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use

I.e:

```
`commit <https://github.com/overhangio/tutor-discovery/commit/e30a78936d63439bde069aeff11960585bd81592>`__
```

Same thing below.